tests/js/JSONFormatBearTest.py

```diff
@@ -1,6 +1,9 @@
-from bears.js import JSONFormatBear
-from coalib.testing.LocalBearTestHelper import verify_local_bear
-
+from queue import Queue
```

Comment:
could you add a newline here to separate system imports from package imports? :)